app/common/application/resources/app.json

```diff
@@ -2,9 +2,9 @@
   "applications": [
       {"name":"APP_REGISTRATION", "url":"/registration", "logo":"images/hospital.png"},
       {"name":"APP_CLINIC", "url":"/clinic", "logo":"images/clinic.png"},
-      {"name":"APP_VITALS", "url":"/vitals", "logo":"images/vitals.png"},
+      {"name":"APP_VITALS", "url":"/vitals", "logo":"images/vitals.png"}/*,
```

Comment:
@valvijo by adding such an attribute, is it sufficient for that use case or more work need to be done for that to work? I mean do we have already mechanisms within our code base to filter those apps based on that added attribute or we need to add those?